src/PactNet.Native/Verifier/PactVerifier.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using System.Collections.Generic;
+using PactNet.Verifier;
+
+namespace PactNet.Native.Verifier
+{
+    /// <summary>
+    /// Pact verifier
+    /// </summary>
+    public class PactVerifier : IPactVerifier
+    {
+        private readonly PactVerifierConfig config;
+        private readonly IDictionary<string, string> verifierArgs;
+
+        /// <summary>
+        /// Initialises a new instance of the <see cref="PactVerifier"/> class.
+        /// </summary>
+        public PactVerifier() : this(new PactVerifierConfig())
+        {
+        }
+
+        /// <summary>
+        /// Initialises a new instance of the <see cref="PactVerifier"/> class.
+        /// </summary>
+        /// <param name="config">Pact verifier config</param>
+        public PactVerifier(PactVerifierConfig config)
+        {
+            
+        }
+
+        /// <summary>
+        /// Initialises a new instance of the <see cref="PactVerifier"/> class.
+        /// </summary>
+        /// <param name="verifierArgs">Pact verifier args</param>
+        /// <param name="config">Pact verifier config</param>
+        internal PactVerifier(IDictionary<string, string> verifierArgs, PactVerifierConfig config)
+        {
+            this.config = config;
+            this.verifierArgs = verifierArgs;
```

Comment:
verifier or config should not be null

Reply:
This is an internal constructor so we know that these won't be null at this point